src/index.js

```diff
@@ -4,17 +4,29 @@ import ColorLib from '@kurkle/color';
 function createGradient(ctx, axis, area) {
   return axis === 'y'
     ? ctx.createLinearGradient(0, area.bottom, 0, area.top)
-    : ctx.createLinearGradient(area.left, 0, area.right, 0);
+    : axis === 'r'
+      ? ctx.createRadialGradient(area.xCenter, area.yCenter, 0, area.xCenter, area.yCenter, area.drawingArea)
+      : ctx.createLinearGradient(area.left, 0, area.right, 0);
 }
 
-function addColors(gradient, scale, colors) {
+function getPixelStop(scale, value) {
+  if (scale.type === 'radialLinear') {
+     const pixel = scale.getDistanceFromCenterForValue(value);
+     const stop = pixel / scale.drawingArea;
+     return {pixel, stop};		
+  }
   const reverse = scale.options.reverse;
+  const pixel = scale.getPixelForValue(value);
+  const stop = scale.getDecimalForPixel(pixel);
+  return {pixel, stop: reverse ? 1 - stop : stop};		
```

Comment:
The reverse check has been move here because it must be calculated for cartesian scales and not for radial, because `getDistanceFromCenterForValue` takes care  about the reverse

Reply:
Ok, I think the `getDecimalForPixel` is flawed for external use cases. Maybe something to consider changing in chart.js v4.